lrlex/src/lib/ctbuilder.rs

```diff
@@ -815,9 +817,15 @@ where
         };
         // Try and run a code formatter on the generated code.
         let unformatted = out_tokens.to_string();
-        let outs = syn::parse_str(&unformatted)
+        let mut outs = String::new();
+        // Record the time that this version of lrlex was built. If the source code changes and rustc
+        // forces a recompile, this will change this value, causing anything which depends on this
+        // build of lrlex to be recompiled too.
+        let timestamp = env!("VERGEN_BUILD_TIMESTAMP");
+        write!(outs, "// lrlex build time: {}\n\n", quote!(#timestamp),).ok();
+        outs.push_str(&syn::parse_str(&unformatted)
             .map(|syntax_tree| prettyplease::unparse(&syntax_tree))
```

Comment:
I think this patch needs `cargo fmt` and fixing of `cargo clippy` errors before it'll pass CI.